vips/vip-520/bsctestnet.ts

```diff
@@ -0,0 +1,161 @@
+- Execution on testnet ([BNB Chain](https://testnet.bscscan.com/tx/0x))`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
+    [
+      // Add Market for asBNB
+      {
+        target: asBNBMarketSpec.vToken.comptroller,
+        signature: "_supportMarket(address)",
+        params: [asBNBMarketSpec.vToken.address],
+      },
+      {
+        target: asBNBMarketSpec.vToken.comptroller,
+        signature: "_setMarketSupplyCaps(address[],uint256[])",
+        params: [[asBNBMarketSpec.vToken.address], [asBNBMarketSpec.riskParameters.supplyCap]],
+      },
+      {
+        target: asBNBMarketSpec.vToken.comptroller,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[asBNBMarketSpec.vToken.address], [asBNBMarketSpec.riskParameters.borrowCap]],
+      },
+      {
+        target: asBNBMarketSpec.vToken.address,
+        signature: "setAccessControlManager(address)",
+        params: [bsctestnet.ACCESS_CONTROL_MANAGER],
+      },
+      {
+        target: asBNBMarketSpec.vToken.address,
+        signature: "setProtocolShareReserve(address)",
+        params: [PROTOCOL_SHARE_RESERVE],
+      },
+      {
+        target: asBNBMarketSpec.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [REDUCE_RESERVES_BLOCK_DELTA_BSC],
+      },
+      {
+        target: asBNBMarketSpec.vToken.address,
+        signature: "_setReserveFactor(uint256)",
+        params: [asBNBMarketSpec.riskParameters.reserveFactor],
+      },
+      {
+        target: asBNBMarketSpec.vToken.comptroller,
+        signature: "_setCollateralFactor(address,uint256)",
+        params: [asBNBMarketSpec.vToken.address, asBNBMarketSpec.riskParameters.collateralFactor],
+      },
+      {
+        target: asBNBMarketSpec.vToken.underlying.address,
+        signature: "faucet(uint256)",
+        params: [asBNBMarketSpec.initialSupply.amount],
+      },
+      {
+        target: asBNBMarketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [asBNBMarketSpec.vToken.address, asBNBMarketSpec.initialSupply.amount],
+      },
+      {
+        target: asBNBMarketSpec.vToken.address,
+        signature: "mintBehalf(address,uint256)",
+        params: [bsctestnet.NORMAL_TIMELOCK, asBNBMarketSpec.initialSupply.amount],
+      },
+      {
+        target: asBNBMarketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [asBNBMarketSpec.vToken.address, 0],
+      },
+      {
+        target: asBNBMarketSpec.vToken.address,
+        signature: "transfer(address,uint256)",
+        params: [asBNBMarketSpec.initialSupply.vTokenReceiver, asBNBMarketSpec.initialSupply.vTokensToBurn],
```

Comment:
For burning we can transfer to `ethers.constants.AddressZero`

Reply:
Yes, but ticket specified `vTreasury` as "burn" tokens receiver on testnet: https://github.com/VenusProtocol/vips/pull/577#discussion_r2144514092